template.yaml

```diff
@@ -89,6 +93,13 @@ Conditions:
     - !Equals [!Ref DeployAlarmsInDev, true]
   EnableSpotArmInstance: !Equals [!Ref Environment, dev]
 # DO WE NEED TO CHANGE THE ARN?
+  UsePermissionsBoundary: !Not
```

Comment:
There is already a reference to 'UsePermissionBoundary' on line 80, better to remove one of those to avoid duplication and potential issues this may cause

Reply:
Good find @sandrasoi. made the changes.